src/scenic/__main__.py

```diff
@@ -98,13 +107,16 @@
         except ValueError:
             pass
     params[name] = value
-translator.dumpTranslatedPython = args.dump_initial_python
+translator.dumpScenicAST = args.dump_scenic_ast
 translator.dumpFinalAST = args.dump_ast
 translator.dumpASTPython = args.dump_python
 translator.usePruning = not args.no_pruning
-if args.seed is not None and args.verbosity >= 1:
-    print(f'Using random seed = {args.seed}')
+if args.seed is not None:
+    if args.verbosity >= 1:
+        print(f'Using random seed = {args.seed}')
+
     random.seed(args.seed)
+    numpy.random.seed(args.seed)
```

Comment:
This is a good idea; let's mention in the documentation of the `seed` option that it also sets the NumPy seed (since that's a change in 3.0 and people might already be setting the seed themselves).